.github/workflows/main.yaml

```diff
@@ -0,0 +1,54 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Test executor inputs
+
+on:
+  push:
+    branches: [main, develop]
+  pull_request:
+    branches: [main, develop]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Clone zkevm-rom
+        uses: GuillaumeFalourd/clone-github-repo-action@v2
+        with:
+          owner: '0xPolygonHermez'
+          repository: 'zkevm-rom'
+      - name: setup zkevm-rom
+        run: |
+          cd zkevm-rom          
+          npm i
+          npm run build
+      - name: Clone zkevm-testvectors
```

Comment:
Which version it is downloaded? which branch?
Could we configure it ?

Reply:
Now last commit of main is downloaded, is a git clone. We can configure the branch with a checkout at the run statement

Reply:
Maybe we could set `branch name` as a environment variable to checkout in GHA script